src/OrchardCore.Modules/OrchardCore.ContentFields/Settings/UserPickerFieldSettingsDriver.cs

```diff
@@ -24,17 +25,16 @@ public override IDisplayResult Edit(ContentPartFieldDefinition partFieldDefiniti
             model.Hint = settings.Hint;
             model.Required = settings.Required;
             model.Multiple = settings.Multiple;
-            var roles = (await _roleService.GetRoleNamesAsync())
-                .Except(RoleHelper.SystemRoleNames, StringComparer.OrdinalIgnoreCase)
-                .Select(roleName => new RoleEntry
-                {
-                    Role = roleName,
-                    IsSelected = settings.DisplayedRoles.Contains(roleName, StringComparer.OrdinalIgnoreCase)
-                })
-                .ToArray();
+            var roles = await _roleService.GetRolesAsync();
+            var roleEntries = roles.Where(role => role.Type != RoleType.System)
+            .Select(role => new RoleEntry
+            {
+                Role = role.RoleName,
+                IsSelected = settings.DisplayedRoles.Contains(role.RoleName, StringComparer.OrdinalIgnoreCase)
+            }).ToArray();
```

Comment:
My format correct. VS does starts the indentation at the same line as v in var.

Reply:
VS is wrong, then. You've used the same pattern I'm suggesting above e.g. here: https://github.com/OrchardCMS/OrchardCore/pull/16781/files#diff-0988673e4e89787687a6344753cd062d103d7a1237a42b3f10cd1fb7a5e7e24d

Reply:
I don't argue with VS when it comes to spacing. I just hit enter, and write my code when it tells me to place my code. So I rather no change it and trust VS "assuming VS is good".

Reply:
This is the indentation pattern we use throughout the codebase. You yourself used it in this PR. It makes more sense because the second line is not a variable declaration or a statement, but a continuation of the variable declaration.

Reply:
I am not arguing which one is better. I trust VS's indentation. If I change it, then I am adding my own indentation which is what I don't want to do. If you look in the codebase, you'll see inconsistency because we don't just following what VS suggests. in all of my PR's the code submitted is based on VS suggestions not my style this way its not a debate or inconsistent.

Reply:
Maybe VS itself doesn't know what it's doing.

https://github.com/user-attachments/assets/e2c3dd5f-15c2-40dd-b2a3-7787b8b9d4bf

So, unless we deliberately want inconsistency, we can't rely on it.

Reply:
I don't think VS is randomly picking indentation just to mess with us :). I would think VS is following some rules (Indent based on multiple factors.) At the end of the day it would follow same logic everywhere.


In the first and second example from your video it starts a new line when is pattern if found
```
var variable = await ....
```

in the third example, since you are not saving the result into a variable so it offset the by 4 spaces.

In the last example, maybe because it is not using `await` so it indented by 4 spaces.

I don't think we should debate this here. Maybe we should figure out how does VS handles indentation and stick with it. (if we find an issue, maybe we can file a issue with the VS team and see if that can be improved or see if we can set rules that it can follow.

Reply:
Or maybe VS just has bugs and ommissions, as I'm sure you encounter every day too, including all the mistaken indentation changes that I've pointed out under your PRs that VS did, most recently under the External Authentication PR.

https://github.com/user-attachments/assets/dd82b1ac-c0ca-46d1-a8cc-e302deff7deb

It's inconsistent because the lambda throws it off, as it happens not infrequently. And thus when it produces something less readable, instead of sticking to it we should use our best judgement on what works for us, and just click the "Commit suggestion" button ;).

Reply:
wasted lots of time again with silly formatting :) I am just going to bite the bullet as I have more important things than formatting that I need to worry about.